docs/adr/adr_zxing_replacement.md

```diff
@@ -12,43 +12,82 @@ Proposed
 
 ## Context or Problem Statement
 
-The current library used by the frontend to parse QR Codes has been in maintenance mode for the past 2 years. It would be ideal to move the QR Code reading to library that is still actively worked on, so as not to run into issues as devices continue to change and older software gets more and more out of date.
+QR code generation and scanning is a core part of the user workflows for warehousing and inventory management in Boxtribute. In particular, the in-house QR code scanner has a single component in the project, that is referenced in two separate contexts around the application, once of which is the landing page of mobile users. However, the current library set used by the frontend to parse QR Codes ([@zxing/library](https://www.npmjs.com/package/@zxing/library?activeTab=readme) and [@zxing/browser](https://www.npmjs.com/package/@zxing/browser)) has been in maintenance mode for the past 2 years.
+
+This creates a medium-high risk for both active field workers and the development team that increases with time as devices continue to change and older software becomes less and less compatible.
+
+For active field workers, the risk with increasing incompatiblity is if the QR scanner functionality breaks, their ability to effectively receive and distribute aid is compromised. For the development team, since QR scanning is a core application workflow an emergency fix will most likely be needed, which will impact the backlog and timelines of other active projects.
+
+Apart from this, more modern packages could bring secondary benefits, such as smaller bundle sizing (making load times and network usage decrease), quicker detection, and less battery utilization through lower level optimizations.
 
 ## Decision Drivers
 
-Maintaining QR Code reader functionality, increasing longevity and fixing some rare issues that already occure on certain devices.
+1. Reliability
```

Comment:
@MomoRazor since we're talking about size now please change the title to "Reliability and Performance"